content/en/agent/troubleshooting/send_a_flare.md

```diff
@@ -142,7 +142,7 @@ Manually obtain this file and provide it to support if there are any issues with
 ### Kubernetes
 To obtain the archive file in Kubernetes, use the kubectl command:
 ```
-kubectl cp datadog-<pod-name>:/tmp/datadog-agent-<date-of-the-flare>.zip flare.zip
+kubectl cp flare.zip datadog-<pod-name>:/tmp/datadog-agent-<date-of-the-flare>.zip -c agent
```

Comment:
```suggestion
kubectl cp datadog-<pod-name>:/tmp/datadog-agent-<date-of-the-flare>.zip flare.zip -c agent
```
Right? The first file is where you're copying from (your pod) and the second is where to (your machine so you can send it to support)?

Reply:
Thank you, Kari! I thought the same thing when I verified but the above command didn't work, and so I think we need to swap the order as I did. Could you please verify on your end to be sure?

Reply:
I tested again on my sandbox.
I notified the following things.
- The command I suggested didn't work either (I believe it worked when I verified for the first time but this won't be a correct command.)
- Your above command didn't work and I got the following error.
`"tar: removing leading '/' from member names"`
So, I removed "/" from "/tmp/datadog-agent-<date-of-the-flare>.zip" as follows, and this worked properly.

`kubectl cp datadog-<pod-name>:tmp/datadog-agent-<date-of-the-flare>.zip flare.zip -c agent`

I referred to this [issue](https://github.com/kubernetes/kubernetes/issues/58692). 

Thanks,
Ryotaro

Reply:
```suggestion
kubectl cp datadog-<pod-name>:tmp/datadog-agent-<date-of-the-flare>.zip flare.zip -c agent
```